gem_city.rb

```diff
@@ -0,0 +1,50 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+
+  def initialize
+    @people = {
+      thieves: 5,
+      officers: 1
+    }
```

Comment:
Do these two values really need to be joined into Hash? Consider the simplification if they weren't.